src/Leaflet/Controls/SearchEngine.js

```diff
@@ -1084,7 +1093,7 @@ define([
         *
         * @private
         */
-        _setMarker : function (position, information, display) {
+        _setMarker : function (position, information, display, icon) {
```

Comment:
Le paramètre "icon" est ici sensé être un identifiant de marker par défaut '"blue", "orange", "red" and "green"). Le terme "icon" n'est pas forcément adapté. 
Il manque d'ailleurs le rajout du paramètre dans l'entête de la jsDoc de la fonction _setMarker().

Reply:
Fait